domains/certificates/Query.API/API/Query.API/ApiModels/Responses/Certificate.cs

```diff
@@ -46,4 +53,10 @@ public class Certificate
     /// </summary>
     public string FuelCode { get; set; } = "";
 
+    /// <summary>
+    /// The type of the certificate. Can be either Production, Consumption or Invalid
+    /// </summary>
+    [JsonConverter(typeof(JsonStringEnumConverter))]
+    [JsonPropertyName("certificateType")]
```

Comment:
why specify the json property name here?

Reply:
So that a rename does not make breaking contract changes

Reply:
We have a test that breaks if there is any changes to the contract. I cannot see that this is needed before someone actually does a rename of the property and do not want a change of API contract.

Reply:
Ok - deleted